publish/models/version.py

```diff
@@ -29,6 +29,9 @@ class Version(models.Model):
         default=_get_default_version,
     )  # TODO: rename this?
 
+    name = models.TextField()
```

Comment:
I would expect `name` to be shorter, so maybe a `CharField` is more appropriate?

Reply:
If we use a `CharField`, we'd need a `max_length`.

@mgrauer @AlmightyYakob Does the schema say anything about the length of these fields? I think we need some sort of limit, for basic security considerations.

Reply:
150 for name, 3000 for description. Schema doesn't say anything but I checked with Satra and Yarik, we agreed to these limits.